src/PluginSuppression.php

```diff
@@ -407,7 +407,7 @@ public function render_suppressed_plugins() {
 										?>
 									<?php endif; ?>
 
-									<?php if ( version_compare( $suppressed_plugins[ $plugin_slug ][ Option::SUPPRESSED_PLUGINS_LAST_VERSION ], $plugin['Version'], '!=' ) ) : ?>
+									<?php if ( $suppressed_plugins[ $plugin_slug ][ Option::SUPPRESSED_PLUGINS_LAST_VERSION ] !== $plugin['Version'] ) : ?>
```

Comment:
The issue is that `$suppressed_plugins[ $plugin_slug ]` returns `false`; in PHP < 7.4 accessing a non-existent index would return `null`, but on PHP 7.4 this now throws an error.

Reply:
I had some more logic errors in the tests. I believe this is fixed now with dd9b9b35d.